### PR TITLE
Separate McNemar's test and contingency table into separate metrics

### PR DESCRIPTION
**Pull request by alex-golts**

### CODE REVIEW

**Review by mosheraboh — CHANGES_REQUESTED** [review]
Hi Alex.
All the pull requests should go into fuse2.
It should be effortless to move it to be on top of fuse2 cause nothing was changed in the eval package.

**Comment by alex-golts** [comment]
@mosheraboh I thought that *because* nothing was changed in eval, it made sense to bring this change quicker to master, no? otherwise external users will wait a long time to use it, till we finish ALL fuse2 porting... you don't agree?

**Review by mosheraboh — COMMENTED** [review]
I agree, 
But in that case, after you merge to master,
Please also go to fuse2, pull those changes, and make sure that all tests pass.
I will review it later today.

File: fuse/eval/metrics/classification/metrics_model_comparison_common.py [hunk 1]
```diff
@@ -36,17 +37,27 @@ def __init__(self, pred1: str, pred2: str, target: str, class_names: Optional[Se
...
-                 class_names: Optional[Sequence[str]] = None, **kwargs):
+    def __init__(self, pred1: str, pred2: str, target: Optional[str]=None, exact: Optional[bool]=True, **kwargs):
         """
-        McNemar's statistical test for comparing two models' predictions in the sense 
-        of the statistics of their disagreements, as seen in the contingency table.
-        :param cls_pred1: key name for the class predictions of model 1
-        :param cls_pred2: key name for the class predictions of model 2
-        :param class_names: class names. required for multi-class classifiers
-        :param sample_weight: optional sample weights. not implemented at this time
+        McNemar's statistical test for comparing two model's predictions or accuracies
```
[the remark this answers is not in the document]
Comment (alex-golts) [reply fuse/eval/metrics/classification/metrics_model_comparison_common.py:54]: done

**Pull request merged**